content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -2936,6 +2936,48 @@ model User {
 </tab>
 </TabbedContent>
 
+### <inlinecode>sequence()</inlinecode>
+
+<Admonition type="warning">
+
+**Only supported by CockroachDB** <br />
+The sequence function is for now only supported by [CockroachDB connector](/concepts/database-connectors/cockroachdb).
+
+</Admonition>
+
+Create a sequence of integers in the underlying database and assign the incremented values to the ID values of the created records based on the sequence.
+
+#### Optional arguments
+
+| Argument    | Example                                               | Description                                                                                                                                                                                                                                           |
```

Comment:
this table doesn't look very good rendered...

Reply:
Indeed, out design can not handle that very well :/ 

2 ideas:

1. Remove the "unrelated" arguments from the examples, will make some ~10 characters shorter and make the second column less wide that way.
2. We could combine "Example" and "Description" in one column, and then have the "Example" as the first line always with code highlighting, then another section of the "Description". That would make both readable at the cost of less clear separation - but then at least the description was readable.